shared_memory_dict/dict.py

```diff
@@ -157,7 +157,9 @@ def _get_or_create_memory_block(
         try:
             return SharedMemory(name=name)
         except FileNotFoundError:
-            return SharedMemory(name=name, create=True, size=size)
+            shm = SharedMemory(name=name, create=True, size=size)
+            shm.buf[:] = self._serializer.dumps({})
+            return shm
```

Comment:
Can you write a test to check that log will not be printed when memory is fresh and empty?